catshadow/testdata/catshadow.toml

```diff
@@ -134,7 +179,8 @@ canqB2SXsVTgcF6mDE054jd5NHu4om+DikAkiijippO2WfVKXjyk3KkU/6+7tWun
 J36qY23I6qGScGjq4tPu3w==
 -----END Kyber768-X25519 PUBLIC KEY-----
 """
-      IdentityPublicKeyPem = """-----BEGIN ED25519 SPHINCS+ PUBLIC KEY-----
+    IdentityPublicKeyPem = """
```

Comment:
You are adding cosmetic changes here like these extra carriage returns. This does nothing for your bottom line. Next time focus on making the PR smaller.

Reply:
Your comment adds nothing